doc/manual/expressions/language-constructs.xml

```diff
@@ -177,13 +177,19 @@ map (concat "foo") [ "bar" "bla" "abc" ]</programlisting>
   <listitem><para>An <literal>@</literal>-pattern provides a means of referring
   to the whole value being matched:
 
-<programlisting>
-args@{ x, y, z, ... }: z + y + x + args.a</programlisting>
+<programlisting> args@{ x, y, z, ... }: z + y + x + args.a</programlisting>
+
+but can also be written as:
+
+<programlisting> { x, y, z, ... } @ args: z + y + x + args.a</programlisting>
 
   Here <varname>args</varname> is bound to the entire argument, which
```

Comment:
I don't understand what this means: `args is bound to the entire argument, which is further matched against the pattern { ... } `